source/extensions/filters/network/dubbo_proxy/router/router_impl.h

```diff
@@ -32,6 +32,11 @@ class Router : public Tcp::ConnectionPool::UpstreamCallbacks,
 
   FilterStatus onMessageDecoded(MessageMetadataSharedPtr metadata, ContextSharedPtr ctx) override;
 
+  // DubboFilter::EncoderFilter
+  void setEncoderFilterCallbacks(DubboFilters::EncoderFilterCallbacks& callbacks) override;
+
+  FilterStatus onMessageEncoded(MessageMetadataSharedPtr metadata, ContextSharedPtr ctx) override;
```

Comment:
Does this function implement `DubboFilter::EncoderFilter` too? Perhaps it could be grouped together with `setEncoderFilterCallbacks()` by removing the empty line 37.